scenes/main-menu/menu.tscn

```diff
@@ -6,16 +6,22 @@
 script = ExtResource("c04d5e79836b")
 
 [node name="Combat" type="Button" parent="."]
+offset_left = 50.0
+offset_top = 50.0
+offset_right = 50.0
+offset_bottom = 50.0
 text = "Start Combat"
-min_size = Vector2(100, 50)
-position = Vector2(50, 50)
 
 [node name="WakaWaka" type="Button" parent="."]
+offset_left = 50.0
+offset_top = 100.0
+offset_right = 50.0
+offset_bottom = 100.0
 text = "WAKA WAKA WAKA"
-min_size = Vector2(100, 50)
-position = Vector2(50, 100)
 
 [node name="Exit" type="Button" parent="."]
+offset_left = 50.0
+offset_top = 800.0
+offset_right = 50.0
+offset_bottom = 800.0
 text = "Exit"
-min_size = Vector2(100, 50)
-position = Vector2(50, 800)
```

Comment:
So every time I open the project, Godot changes these to offsets. Not sure how else to deal with it other than let Godot have its way?